gigareview/frontend/views/manuscript/_form.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use yii\helpers\Html;
+use yii\widgets\ActiveForm;
+
+/** @var yii\web\View $this */
+/** @var common\models\Manuscript $model */
+/** @var yii\widgets\ActiveForm $form */
+?>
+
+<div class="manuscript-form">
+
+    <?php $form = ActiveForm::begin(); ?>
+
+    <?= $form->field($model, 'doi')->textInput() ?>
+
+    <?= $form->field($model, 'manuscript_number')->textInput(['maxlength' => true]) ?>
+
+    <?= $form->field($model, 'article_title')->textInput(['maxlength' => true]) ?>
+
+    <?= $form->field($model, 'publication_date')->textInput() ?>
+
+    <?= $form->field($model, 'editorial_status')->textInput(['maxlength' => true]) ?>
+
+    <?= $form->field($model, 'editorial_status_date')->textInput() ?>
+
```

Comment:
@pli888 The form could not be updated unless the `Editorial Status Date` changes to `07/06/2022`, I think the reason for this is because in the date format of this column is restricted to `dd/MM/yyyy` in `gigareview/common/models/Manuscript.php`.

Reply:
The rule for `editorial_status_date` (as well as for `publication_date`) has been updated so that the dates from EM Manuscript reports are in the form of MM/dd/yyyy.